src/App.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
-import HelloWorld from './Components/HelloWorld/HelloWorld'
+import UserClick from './Components/CapturingUserClick/CapturingUserClick'
 function App() {
 
   return (
     <>
-    <HelloWorld/>
+    <UserClick/>
     </>
```

Comment:
Since you are returning a single component, no need of fragment tags! you can even return on the single line if you like! or you just remove the fragment tags.